pkg/v1/providers/ytt/03_customizations/02_avi/ako-deployment.lib.yaml

```diff
@@ -0,0 +1,255 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: ako-crb
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: ako-cr
+subjects:
+  - kind: ServiceAccount
+    name: ako-sa
+    namespace: tkg-system
+#@ end
+
+
+#@ def ako_statefulset():
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: ako
+  namespace: tkg-system
+  labels:
+    app.kubernetes.io/name: #@ ako_name()
+    app.kubernetes.io/version: #@ akoRepo.images.loadBalancerAndIngressServiceImage.tag
+spec:
+  replicas: 1
+  serviceName: ako
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: #@ ako_name()
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: #@ ako_name()
+    spec:
+      serviceAccountName: ako-sa
+      securityContext: {}
+      containers:
+        - name: #@ ako_name()
+          securityContext: null
+          image: #@ ako_image_with_tag()
+          imagePullPolicy: #@ data.values.AVI_AKO_IMAGE_PULL_POLICY
+          env:
+            - name: CTRL_IPADDRESS
+              valueFrom:
+                configMapKeyRef:
+                  name: avi-k8s-config
+                  key: controllerIP
+            - name: CNI_PLUGIN
+              valueFrom:
+                configMapKeyRef:
+                  name: avi-k8s-config
+                  key: cniPlugin
+            - name: FULL_SYNC_INTERVAL
+              valueFrom:
+                configMapKeyRef:
+                  name: avi-k8s-config
+                  key: fullSyncFrequency
+            - name: CLOUD_NAME
+              valueFrom:
+                configMapKeyRef:
+                  name: avi-k8s-config
+                  key: cloudName
+            - name: CLUSTER_NAME
+              valueFrom:
+                configMapKeyRef:
+                  name: avi-k8s-config
+                  key: clusterName
+            - name: DISABLE_STATIC_ROUTE_SYNC
+              valueFrom:
+                configMapKeyRef:
+                  name: avi-k8s-config
+                  key: disableStaticRouteSync
+            - name: DEFAULT_ING_CONTROLLER
+              valueFrom:
+                configMapKeyRef:
+                  name: avi-k8s-config
+                  key: defaultIngController
+            - name: SEG_NAME
+              valueFrom:
+                configMapKeyRef:
+                  name: avi-k8s-config
+                  key: serviceEngineGroupName
+            - name: VIP_NETWORK_LIST
+              valueFrom:
+                configMapKeyRef:
+                  name: avi-k8s-config
+                  key: vipNetworkList
+            - name: AKO_API_PORT
+              valueFrom:
+                configMapKeyRef:
+                  name: avi-k8s-config
+                  key: apiServerPort
+            - name: SERVICE_TYPE
+              valueFrom:
+                configMapKeyRef:
+                  name: avi-k8s-config
+                  key: serviceType
+            - name: LOG_FILE_PATH
```

Comment:
I think we also don't need this `LOG_FILE_PATH` and `LOG_FILE_NAME`

Reply:
Removed